sandbox.html

```diff
@@ -1,52 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Vue.js Study Jam</title>
 </head>
-<body>
-    <h1>Welcome to the Study Jam - How nice of you to join!!</h1>
-    <div id="app">
-        <h1 v-if="message.length % 2 == 0">{{ message }}</h1>
-        <p v-else>There is nothing</p>
 
-        <button @click="show = !show">Toggle List</button>
-        <button @click="list.push(list.length + 1)">Push Number</button>
-        <button @click="list.pop()">Pop Number</button>
-        <button @click="list.reverse()">Reverse List</button>
+<body>
 
-        <ul v-if="show && list.length">
-            <li v-for="item of list">{{ item }}</li>
+    <div id="app">
+        <h1>{{message}}</h1>
+        <label for="">Enter characters name: </label>
+        <input v-model="txt" type="text" placeholder="Please enter name">
```

Comment:
v-model is a two-way data binding mechanism it's similar as binding `value` attribute and checking the input event  - it's a good directive used when dealing with forms